backend/accounts.go

```diff
@@ -257,12 +257,31 @@ func (backend *Backend) nextAccountNumber(coinCode coinpkg.Code, keystore keysto
 		return account.Configurations.ContainsRootFingerprint(rootFingerprint)
 	}
 
-	nextAccountNumber := uint16(len(backend.filterAccounts(accountsConfig, filter)))
+	nextAccountNumber := uint32(0)
```

Comment:
`int` would've worked just fine here. like I was saying, specific int types get in the way of readability. plus, you convert it to `uint16` in the end anyway. but, `AccoutNumber` returns `uint32`, unfortunately :(

so, you see, different conversions for not benefit.